core/src/main/java/org/bitcoinj/coinjoin/callbacks/RequestKeyParameter.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2023 Dash Core Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.bitcoinj.coinjoin.callbacks;
+
+import org.bitcoinj.wallet.Wallet;
+import org.bouncycastle.crypto.params.KeyParameter;
+
+public interface RequestKeyParameter {
+    KeyParameter requestKeyParameter(Wallet wallet);
+}
```

Comment:
This call back will obtain the key parameter (encryption key) for the wallet.  used with `SendRequest`s.